config/types/filesystem.go

```diff
@@ -101,3 +106,51 @@ func (m Mount) ValidateDevice() report.Report {
 	}
 	return r
 }
+
+func (m Mount) ValidateLabel() report.Report {
```

Comment:
It looks like (most) other validation functions in here have corresponding tests, but this new one doesn't.